sdk/schemaregistry/schema-registry-avro/test/utils/env.browser.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
is there a reason for duplicating code instead of using the one from test-utils?

Reply:
For some reason, the `env` exported by the recorder doesn't work, at least without instantiating the recorder. I do not need to use the latter so I figured I could just have my own env implementation.